nima/http2/webclient/src/main/java/module-info.java

```diff
@@ -14,10 +14,20 @@
  * limitations under the License.
  */
 
+import io.helidon.common.features.api.Feature;
+import io.helidon.common.features.api.HelidonFlavor;
+
 /**
  * HTTP/2 WebClient.
  */
+@Feature(value = "Http2Webclient",
```

Comment:
If we move this under webclient, then the feature should be named `HTTP/2`

Reply:
Renamed